airflow/models/dag.py

```diff
@@ -573,6 +573,13 @@ def owner(self):
         """
         return ", ".join({t.owner for t in self.tasks})
 
+    @property
+    def allow_future_exec_dates(self):
+        return conf.getboolean(
+            'scheduler',
+            'ALLOW_TRIGGER_IN_FUTURE',
```

Comment:
It's not clear to me what the convention is right now. But there are more code using lower case. So let's make this lower case.

```suggestion
            'allow_trigger_in_future',
```